osmaxx/conversion/management/commands/result_harvester.py

```diff
@@ -41,6 +41,10 @@ def _handle_running_jobs(self):
             self._update_job(job_id=job_id, queue=queue)
 
     def _update_job(self, job_id, queue):
+        if job_id is None:
+            logger.exception("job_id is None, None is not a valid id!")
+            return
```

Comment:
Oops, I kinda assumed an Exception is raised here, but that's not what [`logger.exception`](https://docs.python.org/3.4/library/logging.html#logging.Logger.exception) does. Though, it should only be called from an exception handler (i.e., an `except` clause).

Maybe you meant to just call `logger.error`?